stable/atlantis/templates/statefulset.yaml

```diff
@@ -22,11 +22,20 @@ spec:
       labels:
```

Comment:
It's will be awesome if you can add default command to podTemplate like
command: atlantis
it's will be useful for integration with https://banzaicloud.com/blog/inject-secrets-into-pods-vault/
Thanks

Reply:
please  rename `cmd` to `command`